examples/convert.rs

```diff
@@ -72,6 +72,30 @@ fn main() {
         };
         let msl = msl::write_string(&module, options).unwrap();
         fs::write(&args[2], msl).unwrap();
+    } else if args[2].ends_with(".spv") {
+        use naga::back::spirv;
+
+        let debug_enabled;
+
+        if args.len() == 4 {
+            debug_enabled = args[3].parse().unwrap();
+        } else {
+            debug_enabled = true;
+        }
+
+        let mut parser = spirv::parser::Parser::new(&module, debug_enabled);
+        let spirv = parser.parse(&module);
+
+        let mut bytes: Vec<u8> = vec![];
+        for byte in spirv.iter() {
```

Comment:
@kvark came with another suggestion, because this one doesn't work for fixed-sized arrays:
`let bytes = spirv.iter().fold(Vec::with_capacity(spv.len() * 4), |mut v, w| { v.extend_from_slice(&w.to_le_bytes()); v })`